libs/s25main/Window.h

```diff
@@ -253,7 +253,7 @@ class Window
     virtual void Msg_Group_CheckboxChange(unsigned /*group_id*/, unsigned /*ctrl_id*/, bool /*checked*/) {}
     virtual void Msg_Group_ProgressChange(unsigned /*group_id*/, unsigned /*ctrl_id*/, unsigned short /*position*/) {}
     virtual void Msg_Group_ScrollShow(unsigned /*group_id*/, unsigned /*ctrl_id*/, bool /*visible*/) {}
-    virtual void Msg_Group_OptionGroupChange(unsigned /*group_id*/, unsigned /*ctrl_id*/, int /*selection*/) {}
+    virtual void Msg_Group_OptionGroupChange(unsigned /*group_id*/, unsigned /*ctrl_id*/, unsigned /*selection*/) {}
```

Comment:
wasn't it signed because of "-1" as "no selection"?

Reply:
The docu once said 0xFFFF was "no selection". As you see it was really confusing. There now is no "no selection" as this was a strange corner case anyway... And as "selection" is any ID there is no "invalid marker" left. I started using optional<unsigned> but that got quite messy either for no current use case. Hence the only way to get "no selection" is if you create an option group with no buttons (or don't initially call SetSelection) and then call GetSelection. But the callback can never return "no selection" as it is a callback called when the selection changes which only happens if something else gets selected

Reply:
oh okay. shouldnt the rest be changed too then?

Reply:
What rest? I changed all `OptionGroupChange` messages